--- nautilus-dropbox-0.6.3/dropbox.in~	2010-06-15 02:36:52.000000000 +0300
+++ nautilus-dropbox-0.6.3/dropbox.in	2010-07-08 20:55:18.697946297 +0300
@@ -524,7 +524,10 @@
     return newmeth
 
 def start_dropbox():
-    db_path = os.path.expanduser(u"~/.dropbox-dist/dropboxd").encode(sys.getfilesystemencoding())
+    # first try dropboxd from system, fallback to user install
+    db_path = "/usr/bin/dropboxd"
+    if not os.access(db_path, os.X_OK):
+        db_path = os.path.expanduser(u"~/.dropbox-dist/dropboxd").encode(sys.getfilesystemencoding())
     if os.access(db_path, os.X_OK):
         f = open("/dev/null", "w")
         # we don't reap the child because we're gonna die anyway, let init do it
